Trouble trying to build with Webpack Can't resolve './loader'

Hello,

Im trying to use the chart component in my Webapp.
My Client is written in Typescript and bundled using Webpack.

I have a simple Index view as a Lit Webcomponent to test the chart:

```ts
import '@google-web-components/google-chart';

import { LitElement, html } from 'lit';
import { customElement } from 'lit/decorators.js';

@customElement('index-view')
export class IndexView extends LitElement {
  render() {
    return html`
      <google-chart
        data='[["Month", "Days"], ["Jan", 31], ["Feb", 28]]'
      ></google-chart>
    `;
  }
}
```

When I try to build the Client using Webpack i get:
```
ERROR in ./node_modules/@google-web-components/google-chart/google-chart.js 25:0-57
Module not found: Error: Can't resolve './loader' in '/home/sknobloch/Documents/Programming/dashboard/node_modules/@google-web-components/google-chart'
```

Not sure if the problem is with the library or my Webpack config.
A hacky/temporary fix is editing the `google-chart.js` and adding `.js` to the import for the loader.
This works for testing but not building via CI/CD.

Here is the `webpack.config.js` i use:
```js
// eslint-disable-next-line @typescript-eslint/no-var-requires
const MiniCssExtractPlugin = require('mini-css-extract-plugin');
// eslint-disable-next-line @typescript-eslint/no-var-requires
const HtmlWebpackPlugin = require('html-webpack-plugin');
// eslint-disable-next-line @typescript-eslint/no-var-requires
const path = require('path');
// eslint-disable-next-line @typescript-eslint/no-var-requires
const TsconfigPathsPlugin = require('tsconfig-paths-webpack-plugin');
// eslint-disable-next-line @typescript-eslint/no-var-requires
const TerserPlugin = require('terser-webpack-plugin');

const webpackConfig = {
  mode: 'development',
  entry: {
    scripts: './apps/client/src/main.ts',
    bundle: './scss/main.scss',
  },
  externals: [],
  devtool: 'source-map',
  output: {
    path: path.resolve(__dirname, 'assets'),
    filename: 'js/[name].[contenthash].js',
    publicPath: '/',
  },
  plugins: [
    new MiniCssExtractPlugin({
      filename: 'css/[name].[contenthash].css',
      chunkFilename: 'css/[name].css',
    }),
    new HtmlWebpackPlugin({
      filename: 'index.html',
      meta: {
        'theme-color': '#AD5B08',
        viewport: 'width=device-width, initial-scale=1.0',
      },
    }),
  ],
  module: {
    rules: [
      {
        test: /\.ts$/,
        loader: 'ts-loader',
      },
      {
        test: /\.(png|svg|jpg|jpeg|gif)$/i,
        type: 'asset/resource',
      },

      {
        test: /.scss$/,
        use: [
          {
            loader: MiniCssExtractPlugin.loader,
          },
          {
            loader: 'css-loader',
            options: {
              sourceMap: true,
            },
          },
          {
            loader: 'resolve-url-loader',
          },
          {
            loader: 'sass-loader',
            options: {
              sourceMap: true,
            },
          },
        ],
      },
      {
        test: /.pug$/,
        loader: 'pug-loader',
      },
    ],
  },
  resolve: {
    extensions: ['.ts', '.js', '.css', '.scss'],
    plugins: [
      new TsconfigPathsPlugin({
        configFile: './apps/client/tsconfig.json',
      }),
    ],
  },
};

if (webpackConfig.mode !== 'development') {
  webpackConfig.plugins.push(new TerserPlugin({ parallel: true }));
}

module.exports = webpackConfig;
```

Thanks for any help!

--- a/google-chart.ts
+++ b/google-chart.ts
@@ -18,7 +18,7 @@
 import {html, css, LitElement} from 'lit';
 import {property} from 'lit/decorators.js';
 
-import {createChartWrapper, dataTable, DataTableLike} from './loader';
+import {createChartWrapper, dataTable, DataTableLike} from './loader.js';
 
 const DEFAULT_EVENTS = ['ready', 'select'];
 

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
     "@web/dev-server": "^0.1.25",
     "@web/test-runner": "^0.13.20",
     "sinon": "^11.1.2",
-    "typescript": "^4.4.4"
+    "typescript": "^4.5.4"
   },
   "typings": "google-chart.d.ts",
   "dependencies": {

